open_academy/models/open_academy_session.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields
+
+
+class OpenAcademySession(models.Model):
+    _name = "open.academy.session"
+    _description = "model of session on open academy"
+
+    name = fields.Char(required=True, size=50)
+    duration = fields.Text()
+    startDate = fields.Datetime()
+    numberSeats = fields.Integer()
+    instructor = fields.Many2one("res.partner", "instructor")
+    course = fields.Many2one("open.academy.course", required=True)
+    attendes = fields.Many2many("res.partner", "attendees")
```

Comment:
```suggestion
    instructor_id = fields.Many2one("res.partner", "instructor")
    course_id = fields.Many2one("open.academy.course", required=True)
    attendee_ids = fields.Many2many("res.partner", "attendees")
```

See guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields